Bootstrap/app/src/main/java/ar/com/wolox/android/example/RootActivity.java

```diff
@@ -0,0 +1,36 @@
+package ar.com.wolox.android.example;
+
+import android.content.Context;
+import android.content.Intent;
+import android.content.SharedPreferences;
+
+import ar.com.wolox.android.R;
+import ar.com.wolox.android.example.ui.home.HomeActivity;
+import ar.com.wolox.android.example.ui.logIn.LogInActivity;
+import ar.com.wolox.wolmo.core.activity.WolmoActivity;
+
+public class RootActivity extends WolmoActivity {
+    @Override
+    protected int layout() {
+        return  R.layout.activity_base;
+    }
+
+    @Override
+    protected void init() {
+        Intent intent;
+        SharedPreferences pref = getSharedPreferences("prefs", Context.MODE_PRIVATE);
+        String username = pref.getString("mail", "");
+        if (username.equals("")){
```

Comment:
You can use the [isEmpty](https://www.geeksforgeeks.org/java-string-isempty-method-example/) of the string object